alarm_req_listener.cpp

```diff
@@ -221,7 +221,12 @@ void AlarmReqListener::listener()
       zmq_clean_sck();
       return;
     }
-
+    // The ZMQ message read here contains the name of the alarm
+    // issuer e.g. "monit" and the alarm identifier e.g. "1000.3"
+    // Note the alarm identifier uses ituAlarmPerceivedSeverity. We can
+    // translate this to alarmModelState using the function 
+    // AlarmTableDef::state this mapping is described in RFC 3877
```

Comment:
"We can translate this to alarmModelState using the function AlarmTableDef::state this mapping is described in RFC 3877 section 5.4" isn't quite grammatical - I'd expect to see something before the "this" (i.e. "We can translate this to alarmModelState using the function AlarmTableDef::state - this mapping is described in RFC 3877 section 5.4" or "We can translate this to alarmModelState using the function AlarmTableDef::state. This mapping is described in RFC 3877 section 5.4").